src/Components/AddColleague/AddColleagueForm/index.jsx

```diff
@@ -1,17 +1,67 @@
 import { useState } from 'react';
-import { Button, Dropdown, Form } from 'semantic-ui-react';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import {
+  Button, Dropdown, Form, Modal,
+} from 'semantic-ui-react';
+import { ROOT_PATH } from '../../../Constants';
+import handleSetColleagueData from '../../../Redux/AddColleagueReducer/Action';
 
 const AddColleagueForm = () => {
   const [username, setUsername] = useState('');
-  const [contact, setContact] = useState('');
+  const [colleagueContact, setContact] = useState(null);
```

Comment:
Maintain consistency in naming. 
```suggestion
  const [colleagueContact, setColleagueContact] = useState(null);
```